test/musica/test_musica_api.F90

```diff
@@ -443,6 +475,26 @@ subroutine test_terminator()
     air_pressure_thickness(:,1) = (/ 900.0_kind_phys, 905.0_kind_phys /)
     air_pressure_thickness(:,2) = (/ 910.0_kind_phys, 915.0_kind_phys /)
 
+    ! Set conditions for one daytime and one nighttime column
+    ! Greenwich, UK and Wellington, NZ
+    latitude = (/ 51.0_kind_phys, -41.0_kind_phys /)
+    longitude = (/ 0.0_kind_phys, 175.0_kind_phys /)
+    earth_eccentricity = 0.0167_kind_phys
+    earth_obliquity = 23.5_kind_phys * DEGREE_TO_RADIAN
+    perihelion_longitude = 102.9_kind_phys * DEGREE_TO_RADIAN
+    moving_vernal_equinox_longitude = 210.0_kind_phys * DEGREE_TO_RADIAN
+    calendar_day = 183.5_kind_phys ! noon GMT Jul 1
+
+    ! Set conditions for one daytime and one nighttime column
+    ! Greenwich, UK and Wellington, NZ
+    latitude = (/ 51.5_kind_phys, -41.3_kind_phys /)
+    longitude = (/ 0.0_kind_phys, 174.8_kind_phys /)
+    earth_eccentricity = 0.0167_kind_phys
+    earth_obliquity = 23.5_kind_phys * DEGREE_TO_RADIAN
+    perihelion_longitude = 102.9_kind_phys * DEGREE_TO_RADIAN
+    moving_vernal_equinox_longitude = 182.7_kind_phys * DEGREE_TO_RADIAN
+    calendar_day = 183.5_kind_phys ! noon GMT Jul 1
+
```

Comment:
These are duplicates (lines 478–486). Could you remove them?

Reply:
thanks for catching this. I removed the duplicate